public/src/pages/auth/Login/Login.tsx

```diff
@@ -74,9 +75,14 @@ export const Login: FC = () => {
         setLoginResponse(data);
         return data;
       })
-      .then(({ email, errorText }) => {
+      .then(async ({ email, errorText }) => {
         setErrorText(errorText);
         sessionStorage.setItem('email', email);
+        const userData = await getUserData(email);
+        sessionStorage.setItem(
+          'userName',
+          `${userData.firstName} ${userData.lastName}`
+        );
```

Comment:
It's a bit odd and messy. Please use either Promise API or async/await statement

```ts
getUser(params, config)
  .then((data: LoginResponse) => {
    setLoginResponse(data);

    return data;
  })
  .then(({ email, errorText }) => {
    if (errorText) {
      setErrorText(errorText);
    }
    
    sessionStorage.setItem('email', email);
    
    return getUserData(email);
  })
  .then((userData) => sessionStorage.setItem(
    'userName',
    `${userData.firstName} ${userData.lastName}`
  ));
```

or

```ts
const data: LoginResponse = getUser(params, config);

setLoginResponse(data);

const { email, errorText } = data;

if (errorText) {
  setErrorText(errorText);
}

sessionStorage.setItem('email', email);

const userData = await getUserData(email);

sessionStorage.setItem(
  'userName',
  `${userData.firstName} ${userData.lastName}`
);
```